diff --git a/drivers/Kconfig b/drivers/Kconfig
index f2bed2ddeb6628..87fca8decbf6de 100644
--- a/drivers/Kconfig
+++ b/drivers/Kconfig
@@ -253,4 +253,7 @@ source "drivers/cdx/Kconfig"
 
 source "drivers/resctrl/Kconfig"
 
+source "drivers/lkcamp/Kconfig"
+
 endmenu
+
diff --git a/drivers/Makefile b/drivers/Makefile
index 0841ea851847e8..8cb7b2f1cb03dc 100644
--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -198,3 +198,5 @@ obj-y				+= resctrl/
 
 obj-$(CONFIG_DIBS)		+= dibs/
 obj-$(CONFIG_S390)		+= s390/
+
+obj-y               += lkcamp/
